src/i18n/strings/en_EN.json

```diff
@@ -21,7 +21,7 @@
     "Whether or not you're using the Richtext mode of the Rich Text Editor": "Whether or not you're using the Richtext mode of the Rich Text Editor",
     "Your homeserver's URL": "Your homeserver's URL",
     "Whether you're using %(brand)s on a device where touch is the primary input mechanism": "Whether you're using %(brand)s on a device where touch is the primary input mechanism",
-    "Whether or not you're using the 'breadcrumbs' feature (avatars above the room list)": "Whether or not you're using the 'breadcrumbs' feature (avatars above the room list)",
+    "Whether or not you're using the 'breadcrumbs' feature (avatars above the room list)": "Whether or not you're using the 'breadcrumbs' feature (avatars above the discussion list)",
```

Comment:
This is changing the default element behaviour.  As discussed on teams, I'd suggest updating the _t function to replace words (e.g. using the /b word boundary https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Assertions) in the translated string based on case sensitive values defined in the config e.g.
translation_word_replacements: [
  "Room": "Discussion",
  "room": "discussion",
  "Chat": "Discussion",
  "chat": "discussion"
]